packages/react/src/useEditor.ts

```diff
@@ -17,7 +17,13 @@ export const useEditor = (options: Partial<EditorOptions> = {}, deps: Dependency
 
     setEditor(instance)
 
-    instance.on('transaction', forceUpdate)
+    instance.on('transaction', () => {
+      requestAnimationFrame(() => {
+        requestAnimationFrame(() => {
```

Comment:
@philippkuehn is there a reason why we need to call `requestAnimationFrame` within a `requestAnimationFrame` ? Why is 1 `requestAnimationFrame` not sufficient?

Reply:
I always do a double RAF because of [this](https://stackoverflow.com/questions/44145740/how-does-double-requestanimationframe-work). Maybe it’s not needed here. Is there any issue with that?